test/testHelpers.js

```diff
@@ -0,0 +1,49 @@
+import sinon from 'sinon';
+import Meteor from '../src/Meteor';
+
+export const stubs = new Map();
+
+export const stub = (target, name, handler) => {
```

Comment:
Why do we need a custom stub or is this just for managing stubs?

Reply:
Yes, this is just a test helper to abstract away the whole sinon code. With this you can simply always do

```js
afterEach(() => {
  restoreAll() // restore all stubs, if any are registered
})
```

so you never need to manually clean up stubs, something I thought to be very annoying in the past